endpointmanager/cmd/endptquerier/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"os"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/config"
+	endptQuerier "github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager/fhirendpointquerier"
+	"github.com/onc-healthit/lantern-back-end/endpoints/fetcher"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager/postgresql"
+)
+
+func failOnError(errString string, err error) {
+	if err != nil {
+		log.Fatalf("%s %s", errString, err)
+	}
+}
+
+func main() {
+	var endpointsFile string
+	if len(os.Args) != 1 {
+		endpointsFile = os.Args[1]
+	} else {
+		failOnError("ERROR: Missing endpoints list command-line argument", nil)
```

Comment:
i don't think this error will occur as expected because the `err` argument is nil

Reply:
This has now been updated to make it clearer.